planning/scenario_planning/scenario_selector/src/scenario_selector_node/scenario_selector_node.cpp

```diff
@@ -39,9 +37,9 @@ void onData(const T & data, T * buffer)
 }
 
 template <class T>
-boost::function<void(const T &)> createCallback(T * buffer)
+std::function<void(T)> createCallback(T * buffer)
 {
-  return static_cast<boost::function<void(const T &)>>(boost::bind(onData<T>, _1, buffer));
+  return static_cast<std::function<void(T)>>(std::bind(&onData<T>, std::placeholders::_1, buffer));
```

Comment:
I think this should be `std::function<void(const T &)>>`, right? Or is it equivalent?

Reply:
Oh i didn't realize this was merged, just nevermind this comment.